setup.py

```diff
@@ -1,10 +1,11 @@
 #!/usr/bin/env python
 
 from setuptools import find_packages, setup
+import pkg_resources
 
 setup(
     name='bootstrap_cfn',
-    version='0.11.2',
+    version=pkg_resources.require("bootstrap-cfn")[0].version,
```

Comment:
Not sure where we're setting the version string? options are, probably easiest to make setup.py the source of truth on the version and just version='x.x.x' here?